### PR TITLE
Fixing a couple minor issues  

### PR DESCRIPTION
**Pull request by CraigLn**
* Adding APNS2 Devices will correctly return the devices that were added.
* Per request configuration changes will correct be applied to the outgoing request.

### CODE REVIEW

File: Sources/PubNub/PubNub.swift [hunk 1]
```diff
@@ -974,11 +1002,16 @@ extension PubNub {
...
     completion: ((Result<[String], Error>) -> Void)?
   ) {
-    route(PushRouter(.manageAPNS(pushToken: deviceToken, environment: environment,
-                                 topic: topic, adding: [], removing: []),
-                     configuration: configuration),
-          responseDecoder: RegisteredPushChannelsResponseDecoder(),
-          custom: requestConfig) { result in
+    route(
+      PushRouter(
+        .manageAPNS(
```
Comment (parfeon) [inline Sources/PubNub/PubNub.swift:1007]: @CraigLn we are inside of `listAPNSPushChannelRegistrations` and it looks like not really correct to use here `manageAPNS`.
Is it `listPushChannels` which should be used here instead?

**Review by parfeon — CHANGES_REQUESTED** [review]
@CraigLn changes mostly are fine, but one change shown potentially an issue in APNS endpoint usage.

File: Sources/PubNub/PubNub.swift (hunk 1 above)
Comment (CraigLn) [reply Sources/PubNub/PubNub.swift:1007]: This is listing APNS2 registrations exclusively.  The server [API REST docs](https://www.pubnub.com/docs/rest-api#push-notifications-apns-http2) seem to indicate it's the same API as Add/Remove:

Comment (parfeon) [reply Sources/PubNub/PubNub.swift:1007]: @CraigLn go it. Then it just confusing enum field name for this type of API.

**Review by parfeon — APPROVED** [review]
LGTM :)

File: Sources/PubNub/PubNub.swift (hunk 1 above)
Comment (CraigLn) [reply Sources/PubNub/PubNub.swift:1007]: The router is an internal concept that is meant to map directly to the PubNub server APIs, so they are not meant to be 100% aligned with the public PubNub instance method names.   A new router enum value would typically only be added when the server adds a new API.

**Pull request merged**